src/styles/blocks/header.scss

```diff
@@ -0,0 +1,31 @@
+//HEADER BLOCK:
+$logo-size: 40px;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
```

Comment:
![image](https://user-images.githubusercontent.com/104728827/185793808-85235872-5662-4102-95dd-768aa931a987.png)
I am not sure you need to set a fixed height